src/test/java/com/forest/mytopmovies/controller/movie/MovieListControllerIT.java

```diff
@@ -258,6 +261,70 @@ void cantDeleteMovieList() throws Exception {
                 .andReturn();
     }
 
+    @Test
+    @Order(7)
+    void cantGetMovieListByIdWithoutToken() throws Exception {
+        // given
+        ObjectMapper mapper = new ObjectMapper();
+        String token = registerUser(mapper, "forest3", "123456");
+
+        String expectedTMDBResponse = FileReaderUtil.readJsonFromFile("src/test/java/com/forest/utils/json_response/tmdb/searchMovieById.json");
+
+        whenHttp(server)
+                .match(Condition.endsWithUri("/movie/550"), Condition.parameter("api_key", "test-key"))
+                .then(ok(), stringContent(expectedTMDBResponse));
+
+        String movieListResponse = createMovieList(mapper, baseMovieListName, baseDescription, baseMovieIds, token);
+        MovieListPojo movieListPojo = mapper.readValue(movieListResponse, MovieListPojo.class);
+
+        // when
+        MvcResult result = this.mockMvc.perform(get("/api/v1/movielist/" + movieListPojo.getId()))
+                .andDo(print())
+                .andExpect(status().isUnauthorized()).andReturn();
+
+        // then
+        assertThat(result.getResponse().getContentAsString()).contains("Missing authentication token");
+    }
+
+    @Test
+    @Order(8)
+    void cantGetMovieListByIdWithExpiredToken() throws Exception {
+        // given
+        ObjectMapper mapper = new ObjectMapper();
+        String token = registerUser(mapper, "forest3", "123456");
+
+        String expectedTMDBResponse = FileReaderUtil.readJsonFromFile("src/test/java/com/forest/utils/json_response/tmdb/searchMovieById.json");
+
+        whenHttp(server)
+                .match(Condition.endsWithUri("/movie/550"), Condition.parameter("api_key", "test-key"))
+                .then(ok(), stringContent(expectedTMDBResponse));
+
+        String movieListResponse = createMovieList(mapper, baseMovieListName, baseDescription, baseMovieIds, token);
+        MovieListPojo movieListPojo = mapper.readValue(movieListResponse, MovieListPojo.class);
+        ScheduledExecutorService executorService = Executors.newSingleThreadScheduledExecutor();
+
+        // when
+
+        // then
```

Comment:
Probably a good idea to put some logging here just so people don't think the test is hanging.